tape/src/main/java/com/squareup/tape2/ObjectQueue.java

```diff
@@ -42,7 +43,7 @@ public boolean isEmpty() {
    * Returns the head of the queue, or {@code null} if the queue is empty. Does not modify the
    * queue.
    */
-  public abstract T peek() throws IOException;
+  public abstract @Nullable T peek() throws IOException;
```

Comment:
Does this return null instead of throwing for parity with other dequeue interfaces?